src/scripts/bttvIntegration.js

```diff
@@ -0,0 +1,105 @@
+function replaceKeywordWithEmoteImageString(textContent, keyword, imageSource, id){ 
+    if (!textContent) { 
+        throw "Text content is not provided!" 
```

Comment:
I'm unclear what would happen if we throw, since I'm not seeing a `catch` anywhere (let me know if I'm missing it!) — would the overlay crash? If so, would it be better to just swallow the error?

Reply:
i was JUST thinking about this. This is out of sync with other files where I simply return null, which seems fine to do for now. especially considering it can help unnecessarily crashing overlay. fixing now